remote_connection.go

```diff
@@ -0,0 +1,83 @@
+package influxdb
+
+import (
+	"context"
+
+	"github.com/influxdata/influxdb/v2/kit/platform"
+)
+
+type RemoteConnectionService interface {
+	// ListRemoteConnections returns all info about registered remote InfluxDB instances matching a filter.
```

Comment:
Nitpick on the comments here: I might say something other than "InfluxDB instances"...maybe "InfluxDB connections" or just "connections", since I don't really think of a connection to cloud as an instance.